stable/redis/templates/redis-slave-deployment.yaml

```diff
@@ -34,6 +34,9 @@ spec:
       {{- end}}
       {{- /* Include master securityContext if slave securityContext not defined */ -}}
       {{ include "redis.slave.securityContext" . | indent 6 }}
+      {{- if .Values.serviceAccount.create }}
```

Comment:
No need to wrap into an if block.

Reply:
@unguiculus Forget about precedent comment. Fixed in cb669279938dc9f22f48118e0ff239a83d6b6e24